markdown/events/2021/bytesize-10-institutional-profiles.md

```diff
@@ -41,3 +41,250 @@ You can see the slides on HackMD: <https://hackmd.io/@jfy133/Hyc_WvXU_> (shown b
+
+But as I said, these profiles can apply to all pipelines, so you can also switch at the bottom to `nf-core mag`, and the same thing will work, but everything will be optimised for that particular cluster.
+
+[4:12](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=252) You can also chain multiple institutional profiles together if you need.
+
+So for example for  `nextflow run nf-core/eager -profile shh,sdag` where we have the dash profile for the MPI in Jena, we have multiple clusters.
+
+So we can specify within that specific cluster the specific institution that you want to use.
+
+Something that’s very important that was mentioned in [Maximes talk](https://nf-co.re/events/2021/bytesize-8-nf-core-offline) is that the order of the profile is very important.
+
+So anything on the furthest right will override anything previously. So any parameters set in the `sdag` profile will override any parameter also set in the `ssh` profile in this given example.
+
+[4:57](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=297) So let’s say you want to write an institutional profile for your cluster, there are multiple things you may want to cover.
+
+[5:05](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=305) And this is the information that you want to gather before you start writing the profile. So it will cover things like names, resource limits on scheduling systems, and also containers.
+
+[5:15](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=315) So the first question you want to ask is what name do you want to call your profile.
+
+This should be something that’s very recognisable and also descriptive so that users of your cluster can recognise which cluster is being referred to.
+
+But it should not be something that is too generic either such that people might confuse it with their own cluster.
+
+Short is also very good, abbreviations are OK, like the `ssh` example from earlier.
+
+But again, it should still be precise enough for people to understand what it is.
+
+If you have multiple clusters for one institution, it might make more sense to have an umbrella name for the institution itself rather than have one profile for each cluster.
+
+Within that you can have internal profiles, which I’ll give you more examples of later on.
+
+So make sure you get the right level of hierarchy there.
+
+[6:13](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=373) Once you decide on a name, you need to go into your cluster and get into information about the specifications of that.
+
+For example, you should look for the resource limits of that particular cluster.
+
+So `nf-core/pipelines` expand on the Nextflow retry system where if a process times out because it runs out of memory or time, it makes it re-submit that job with more memory or time.
```

Comment:
```suggestion
So nf-core pipelines expand on the Nextflow retry system where if a process times out because it runs out of memory or time, it makes it re-submit that job with more memory or time.
```